src/com/lftechnology/java/training/alina/basic/stemleaf/StemLeafRepresentation.java

```diff
@@ -15,76 +15,71 @@
  */
 public class StemLeafRepresentation {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)